RawSpeed/DngDecoderSlices.cpp

```diff
@@ -143,6 +143,159 @@ my_error_throw (j_common_ptr cinfo)
   ThrowRDE("JPEG decoder error!");
 } 
 
+static void decodeFPDeltaRow(unsigned char * src, unsigned char * dst, size_t tileWidth, size_t realTileWidth, unsigned int bytesps, int factor) {
```

Comment:
The GPL and LGPL licenses are not synonyms, they are 2 different things, and i see nothing about LGPL on that page.
But yes, i do think that this MIT code can be used.
Do put `MIT License, copyright 2014 Javier Celaya <jcelaya@gmail.com>` as comment before the function.